salt/metalk8s/container-engine/containerd/installed.sls

```diff
@@ -69,11 +72,15 @@ Create containerd service drop-in:
           HTTPS_PROXY: "{{ proxies.https }}"
           {%- endif %}
         {%- endif %}
+    - watch_in:
+      - service: Ensure containerd running
```

Comment:
Shouldn't this result in a `containerd` restart, and even more, a `systemctl daemon-reload`?

Reply:
Yes it does. Why ?